packages/synapse-react-client/src/components/SchemaDrivenAnnotationEditor/AnnotationEditorUtils.test.ts

```diff
@@ -1,12 +1,28 @@
-import { AjvError } from '@rjsf/core'
+import { RJSFValidationError } from '@rjsf/utils'
```

Comment:
This type was changed in RJSF v5. Had to fix it since this test file was moved to `src/` so is now type-checked